src/Components/SystemDetail/RecommendationsTable.js

```diff
@@ -1,24 +1,61 @@
 import React from 'react';
-import { Table, TableHeader, TableBody, expandable } from '@patternfly/react-table';
+import { Table, TableHeader, TableBody, expandable, TableComposable, Thead, Tr, Th, Td, Tbody } from '@patternfly/react-table';
 import propTypes from 'prop-types';
 import { flatMap } from 'lodash';
 import { EmptyTable } from '@redhat-cloud-services/frontend-components/EmptyTable';
 import { EmptyStateDisplay } from '../EmptyStateDisplay/EmptyStateDisplay';
-import { CheckCircleIcon } from '@patternfly/react-icons';
+import { CheckCircleIcon, BullseyeIcon, WrenchIcon } from '@patternfly/react-icons';
 import { TextContent, Text, TextVariants } from '@patternfly/react-core';
 import './RecommendationsTable.scss';
 
 const renderExpandedView = (row) => {
     return (
         <TextContent>
-            <Text component={TextVariants.p} className="newline tab">
-                <Text><strong>Detected issues</strong></Text>
+            <Text component={TextVariants.p}>
+                <Text className="margin-text-bottom"><BullseyeIcon/><strong className="strong-tag-style">Detected issues</strong></Text>
                 {row.reason}
             </Text>
-            <Text component={TextVariants.p} className="newline">
-                <Text><strong>Suggestion</strong></Text>
+            {row.detected_issues && <TableComposable
+                arial-label="Detected issues table"
+                variant="compact"
+                borders={false}
+                className="table-border-top detected-issues-table"
+            >
+                <Thead>
+                    <Tr>
+                        <Th className="Th-tag-style">Identified issues by ROS</Th>
+                    </Tr>
+                </Thead>
+                <Tbody>
+                    <Tr>
+                        <Td dataLabel="Identified issues by ROS" className="newline tab">{row.detected_issues}</Td>
+                    </Tr>
+                </Tbody>
+            </TableComposable> }
+            <hr/>
+            <Text component={TextVariants.p}>
+                <Text className="margin-text-bottom"><WrenchIcon/><strong className="strong-tag-style">Suggestion</strong></Text>
                 {row.resolution}
             </Text>
+            <TableComposable
```

Comment:
@r14chandra can you add a check here to see it shows table only if we are getting `current_instance` and `suggested_instances` from the API similar to above else it will show the table until backend changes are deployed. Also I would suggest to use destructing on row to extract both values along with `detected_issues`.

![image](https://user-images.githubusercontent.com/5928530/161736640-8183f934-cda1-48b7-92d6-dd26421bd3a1.png)